packages/eth2.0-config/src/index.ts

```diff
@@ -1,6 +1,7 @@
-import {IBeaconParams} from "../params";
+import {IBeaconParams} from "@chainsafe/eth2.0-params";
+import {createIBeaconSSZTypes} from "@chainsafe/eth2.0-ssz-types";
+
```

Comment:
we need method `generateConfig` that will accept Partitial<IBeaconParams> and merge it with mainet preset, generate types and return config object. Presets will only be used in tests

Reply:
Somehow I thought there was already a function `createIBeaconConfig` to match the `createIBeaconSSZTypes`!
will definitely add one.
I think the merge with presets is a detail that should be outside the scope of this module. I'm thinking the implementation looks like:
```typescript
export function createIBeaconConfig(params: IBeaconParams): IBeaconConfig {
  return {
    params,
    types: createIBeaconSSZTypes(params),
  };
}
```
I suspect that parameter preset switching/overriding logic would then live in the cli code.

Reply:
Could be, I kinda like versatile eth2.0-config library that would handle param override. I mean it's a simple deepMerge that would make life easier for anyone using it

Reply:
lets experiment first and revisit once we have a good implementation